### ISSUE TITLE
kill whole line shortcut

### ISSUE THREAD

**Issue by ndr-brt** [yaxu/feedforward#24]
Would be useful to define a shortcut to kill all the line where the cursor is.
At the moment there's the `kill-line` (C-k), but it requires to many digits to accomplish the task.

The keybind could be C-S-k.

**Comment by yaxu** [comment]
Hmm but C-k only requires two keys whereas C-S-k requires three?

**Comment by ndr-brt** [comment]
Yes, but `C-k` kills a line from the cursor to the end of the line (`\n` excluded), I wanted a command like the `dd` in VI, where the current line is deleted completely (`\n` included)

I did some researches/explorations about that and I learned that:
* The emacs keybind for `kill-whole-line` seems to be `C-S-Backspace`, not `C-S-k`
* It's not that easy to catch `C-S` events on ncurses (as you said in #22 )

Enabling logs, I noticed that the `C-S-Backspace` combination generates an `EventCharacter '\b'`, that's generated also with `C-Backspace`, so maybe that could be our shortcut to remove a line.

**Comment by ndr-brt** [comment]
Other information, `C-S-Backspace`, `C-Backspace` and `C-h` launch the same event (`ord` == 8)

**Comment by yaxu** [comment]
FWIW, I use ctrl-a ctrl-k for this

**Comment by yaxu** [comment]
Feedforward has been moved to the tidal repo:
https://github.com/tidalcycles/Tidal/tree/main/feedforward
